theme/static/js/hs_file_browser.js

```diff
@@ -2013,13 +2047,16 @@ function setFileType(fileType){
     var resID = $("#hs-file-browser").attr("data-res-id");
     var url;
     var folderPath = "";
-    if($("#fb-files-container li.ui-selected").hasClass('fb-file')){
-        var file_id = $("#fb-files-container li.ui-selected").attr("data-pk");
+    let selected = $("#fb-files-container li.ui-selected");
+    let name = $("#fb-files-container li.ui-selected").find(".fb-file-name").text().trim();
+
+    if(selected.hasClass('fb-file')){
+        var file_id = selected.attr("data-pk");
```

Comment:
file_id used anywhere  else outside of the if block? How about "let"

Reply:
@engineSound, that is correct. But we probably should update all scripts at some point to ES6 standards. Since right now the use of `var` here doesn't have any implications we should probably avoid fixing these until we are ready to update all of our scripts.